ceph/rbd/workflows/krbd_io_handler.py

```diff
@@ -80,6 +80,10 @@ def krbd_io_handler(**kw):
                 device_names.append(rbd.map(pool=pool_name, image=image_name)[:-1])
 
             if operations.get("device_map"):
+                out = rbd.exec_cmd(cmd="rpm -qa|grep rbd-nbd", sudo=True)
```

Comment:
what is the reason of not checking `output=True` here like below code?

Reply:
That was by mistake, thanks for pointing it out Sunil I have made the changes now.